sass/themes/cr/2019/variables/_typography.scss

```diff
@@ -1,26 +1,27 @@
 // Fonts
-$body-font: "Montserrat", sans-serif;
 $montserrat: "Montserrat", sans-serif;
 $anton: "Anton", sans-serif;
 
+// Backwards compatibility to keep the base theme happy
+$body-font: $montserrat;
 
 /* Sizing variations.
 ** format is as follows: breakpoint: (font-size, line-height)
 */
 
 /* 'Title' H1 variation */
 $h1-title-font-size: (
-  null: (100px, 104px),
+  null: (54px, 62px),
   $screen-md: (100px, 104px),
   $screen-lg: (100px, 104px)
 );
 $h1-font-size: (
-  null: (64px, 72px),
+  null: (42px, 48px),
   $screen-md: (64px, 72px),
   $screen-lg: (64px, 72px)
 );
 $h2-font-size: (
-  null: (42px, 48px),
+  null: (34px, 40px),
```

Comment:
Check against twig file. Is it 32x40 or 34x40?

Reply:
Whoops. SASS right, Twig wrong; have corrected